### PR TITLE
Loss function (GIOU loss) implimented

### PR DESCRIPTION
**Pull request by Ban-Ironic-Ohms**

### CODE REVIEW

**Comment by Ban-Ironic-Ohms** [comment]
@ImTheSquid Take a look?

File: neural/src/giou_tensor.rs
Comment (ImTheSquid) [inline neural/src/giou_tensor.rs:1]: This file probably shouldn't be here. Is there a reason you want to include it?

**Review by ImTheSquid — CHANGES_REQUESTED** [review]

**Comment by Ban-Ironic-Ohms** [comment]
Your right - those were old code files. Entire directory removed as all functionality is captured in the neural dir.

**Review by ImTheSquid — CHANGES_REQUESTED** [review]
Remove all of the lang-translate'd files as well as the java dataset; they blow up the size of the repository and should be stored separately anyways. Also remove neural/src/giou_tensor.rs or tell me why it's needed.

**Comment by Ban-Ironic-Ohms** [comment]
Removed - the lang-translate code itself could still be useful, but we can just go back to rev history and get it if needed

**Comment by ImTheSquid** [comment]
Add back the script you used but not the code that you used the script on

**Comment by Ban-Ironic-Ohms** [comment]
Done

**Review by ImTheSquid — APPROVED** [review]

**Pull request merged**